src/listeners/log-to-console.js

```diff
@@ -23,9 +23,9 @@ export default function(cfg = {}) {
     }
 
     let now = moment(entry._time.stamp).utcOffset(entry._time.utc_offset).toISOString();
-    let levelName = logger.levelToLevelName(entry._level);
+    let levelName = logger.levels.levelToLevelName(entry._level);
     let formattedLevelName = _.padStart(_.toUpper(levelName), '5');
-    let consoleFun = logger.levelToConsoleFun(entry._level);
+    let consoleFun = logger.levels.levelToConsoleFun(entry._level);
```

Comment:
similarly, if someone wants the log-to-console listener to use another levelToConsoleFun, they should pass the fun in the cfg parameter. The point of the PR request was that the logger doesn't care about the console; it's the log-to-console that cares.